config/24.1/plugins.conf

```diff
@@ -25,9 +25,11 @@ net-mgmt/telegraf				arm
 net-mgmt/zabbix-agent@zabbix5			arm
 net-mgmt/zabbix-agent@zabbix6			arm
 net-mgmt/zabbix-agent@zabbix64			arm
+net-mgmt/zabbix-agent@zabbix7			arm
```

Comment:
is there a PR coming? adding these to 24.1 and the packages to 24.7 is going to cause friction ;)